packages/cli-test/src/cli/shell.spec.ts

```diff
@@ -1,9 +1,12 @@
-import { assert } from 'chai';
-import sinon from 'sinon';
 import child from 'child_process';
-import stream from 'stream';
 import EventEmitter from 'events';
+import stream from 'stream';
+
+import { assert } from 'chai';
+import sinon from 'sinon';
+
 import { shell } from './shell';
+
 import type { ShellProcess } from '../utils/types';
```

Comment:
😱 Big fan of splitting stdlib from dependencies from local from types if that's what this rule is requiring!